admin/superuser-init.py

```diff
@@ -1,290 +1,139 @@
-else:
-    print "Group %s already exists" % admin
-    admin_group = client.get_group(admin)
-
-# add admin group to the conference members
-client.add_members_to_group(conf_group, [admin])
-
-# create admin user
-create_admin = raw_input("Create administrator login? (y/[n]): ").lower()
-if create_admin == 'y' or create_admin == 'yes':
-    default_username = conference.split('/')[-1].lower()+'_admin'
-    username = raw_input("Please provide administrator login, in lowercase, with no spaces (default: {0}): ".format(default_username))
-    if not username.strip(): username = default_username
-    firstname = raw_input("Please provide administrator first name: ")
-    lastname = raw_input("Please provide administrator last name: ")
-
-    passwords_match = False
-    while not passwords_match:
-        password = getpass.getpass("Please provide a new administrator password: ")
-        passwordconfirm = getpass.getpass("Please confirm the new password: ")
-
-        passwords_match = password == passwordconfirm
-        if not passwords_match:
-            print "Passwords do not match."
-
-    client.register_user(email = username, password=password,first=firstname,last=lastname)
-
-    manual_activation = raw_input("Would you like to activate the user manually? (y/[n]): ")
-    manual_activation = manual_activation.lower() == 'y'
-
-    if manual_activation:
-        valid_token = False
-        while not valid_token:
-            try:
-                token = raw_input("Please provide the confirmation token: ")
-                activation = client.activate_user(token = token)
-                print "Admin account activated."
-                print "UserID: ", activation['user']['profile']['id']
-                print "Login: ", username
-                valid_token = True
-            except:
-                print "Invalid token."
-    else:
-        print "Admin account not activated. Please respond to the email confirmation sent to %s" % username
-    client.add_members_to_group(admin_group, [username])
-    print "Added %s to %s" % (username, args.conf)
+directory_path = os.path.join(os.path.dirname(__file__), '../venues/{0}'.format(args.conf))
+conference_group_id = args.conf
+
+# load data
+data = parse_json(args.data)
+
+# build the directory structure
+build_directory(directory_path)
+
+templates = [
+    'python/config.template',
+    'python/admin-init.template',
+    'webfield/conferenceWebfield.template',
+    'webfield/programchairWebfield.template',
+    'process/submissionProcess.template',
+    'process/commentProcess.template',
+    'process/officialReviewProcess.template',
+]
+
+# generate new files from templates
+for file in templates:
+    generate_file(file, directory_path, data, overwrite = args.overwrite)
+
+groups = build_groups(conference_group_id)
+
+print "created the following groups:"
+for g in groups: print g
+
+post_groups = raw_input("Post groups to {0}? (y/[n]): ".format(client.baseurl)).lower()
```

Comment:
why did you add this question?

Reply:
so that the user is reminded of which baseurl the client is connected to

do you think it's not needed?

Reply:
I don't know, how is going to run this script, @pmandler ? 

and what happens if you say no? the conference files are created but not the groups? it is weird.

Reply:
ok, fair enough. i'll remove it